### ISSUE TITLE
Revert changes to DPS tests introduced by to-hit audits and re-audit the affected items' damage values to match

### ISSUE THREAD

**Issue by Terrorforge** [CleverRaven/Cataclysm-DDA#60766]
### Is your feature request related to a problem? Please describe.

There has recently been a series of audits to normalize the to-hit values of various weapons, including #59577, #59599, #59656 and #59606. Of course, changing only the to-hit value of changes their DPS, which caused the tests in effective_dps_test.cpp to fail. To fix this problem, it appears that @ANickelN simply reduced the values the tests were expecting to match the new DPS of the weapons.

This was an error. Those values were already calibrated according to the intended weapon balance as laid out in [GAME_BALANCE.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/d465fe418bea7b65290f704c00aed48888136bbb/doc/GAME_BALANCE.md). That's what their dps is supposed to be, regardless of what their to-hit is. As a result of changing the tests instead of the weapons, dozens of weapons are significantly under- or over-performing compared to their category.

### Solution you would like.

Revert all the changes to the tests. Keep the to-hit changes but alter the affected weapons' damage values to match the pre-audit DPS.

### Additional context

This was unearthed thanks to discussion in #60686, as it turned out to be a major reason the Combat Knife deals so much more damage than other fighting knives; it already had the highest dps, and then a bunch of the others lost their to-hit bonus in the audit.

**Comment by x-qq** [comment]
None of the linked four PRs by @ANickelN have touched the combat knife DPS or the tests for its DPS.

**Comment by Terrorforge** [comment]
No, but #59656 significantly decreased the DPS of most of its competitors, thus making the combat knife better by comparison.

It's not terribly relevant to this issue, but fwiw my contention was never that the combat knife is objectively too good, it's that it's way better than every other knife for no good reason. Turns out, it's not supposed to be _that_ much better.

**Comment by x-qq** [comment]
There are two things I would like to point out:
* among the knives, combat knife has basically no competitors in terms of (how readily available it is + DURABLE_MELEE factor). The knives are either not durable (the kitchen stuff, survival knife, etc) or are basically non-existant antique/museum stuff (tanto/kirpan/trench/etc).
* among the other weapon types, its direct competitors (in same terms as above: availability + DURABLE_MELEE) are baseball bat (20 DPS), aluminium bat (21 DPS), machete (20 DPS), kukri (19.9 DPS, its availability is questionable but it has more spawns than other fancy blades), and every single one of them is better than the combat knife (19 DPS).

**Comment by Terrorforge** [comment]
How is that relevant to this issue?

**Comment by x-qq** [comment]
The proposed solution to the issue is to revert the linked 4 patches. I am pointing out that the situation before them was exactly the same and reverting them achieves no practical result.

**Comment by Terrorforge** [comment]
I would encourage you to actually read what I wrote. The combat knife thing is just a side effect, the actual problem is that these weapons no longer follow the guidelines as laid out in the documentation

**Comment by Amoebka** [comment]
The issue adressed by this PR is the blatant discrepancy between the design documents and actual gameplay (and tests). Combat knife is unrelated to this, Terrorforge just mentioned it because he has a personal hangup on the item.

**Comment by Terrorforge** [comment]
Correct.

**Comment by Terrorforge** [comment]
I spoke with ANickel, and it turns out this was not an unintentional mistake. It was a decision arrived at in conjunction with senior devs, chiefly @anothersimulacrum. It was done because, quote:

> They were doing big en-masse changes to a different format, didn't need to get bogged down meticulously adjusting to the perfect values when they can't test it
> I (obviously) think it would be reasonable or someone to go through and shuffle values, but that person either needs to be able to run the tests or be willing to tweak, wait for github, and tweak again

So there is a desire to fix it, it's just kind of a PITA.
